### PR TITLE
Don't redirect from web user edit to my account settings

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?177121

@esoergel / @benrudolph

### CODE REVIEW

File: corehq/apps/users/views/__init__.py [hunk 1]
```diff
@@ -279,8 +279,6 @@ def dispatch(self, request, *args, **kwargs):
         return super(EditWebUserView, self).dispatch(request, *args, **kwargs)
 
     def get(self, request, *args, **kwargs):
-        if self.editable_user_id == self.couch_user._id:
-            return HttpResponseRedirect(reverse(EditMyAccountDomainView.urlname, args=[self.domain]))
         return super(EditWebUserView, self).get(request, *args, **kwargs)
```
Comment (esoergel) [inline corehq/apps/users/views/__init__.py]: this method is a noop now, so you can just delete the whole thing

**Comment by esoergel** [comment]
Looks good, two questions:

1. How does the web user view look/work if you're editing your own user?  Just like you'd expect?
2. I think you should fully remove `EditMyAccountDomainView`.  It's referenced in a few redirects on successful form submission, but there are other reasonable fallbacks for that (might wanna touch base with Amelia on what's appropriate there).

**Comment by orangejenny** [comment]
@esoergel 

Looks like editing any other user. Could change it to say "My Account" in the sidebar menu, but I don't think it's worth the trouble, given that it's now the same page you see for editing any web user.

screen shot 2015-07-27 at 1 47 56 pm

Good point about dropping the view altogether. I'm inclined to redirect back to the list of mobile workers in the cases where we're currently redirecting to the domain-specific my settings, but pinged @amsagoff on the ticket for another opinion.

**Comment by esoergel** [comment]
Awesome, that's a sorta new use-case for that page, so I wanted to make sure it looked reasonable.  I agree that it should just display your email in the sidebar, like any other mobile worker, as you're pointedly NOT editing your account - just your domain membership.

File: corehq/apps/users/views/__init__.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/users/views/__init__.py]: :+1:

**Comment by benrudolph** [comment]
looks good to me. not to familiar with the UI/UX of this particular feature, but the code seems fine. +1 to ethan's comment

**Comment by orangejenny** [comment]
@esoergel Dropped EditMyAccountDomainView, should be g2g now. Looked at the redirects with @amsagoff and it looks like they're for use cases where you're editing/verifying/deleting your own phone number from the view that's being deleted...so dropping the view means those redirects can also be removed.

**Comment by esoergel** [comment]
Looks good to me.
`+7 -66` is always fun.

**Pull request merged**